Kickstarter-iOS/Views/Controllers/DeprecatedRewardPledgeViewController.swift

```diff
@@ -243,14 +242,15 @@ internal final class DeprecatedRewardPledgeViewController: UIViewController {
       |> UILabel.lens.textColor .~ UIColor.ksr_text_dark_grey_400
       |> UILabel.lens.textAlignment .~ .center
       |> UILabel.lens.numberOfLines .~ 2
-      |> UILabel.lens.text %~ { _ in Strings.Kickstarter_is_not_a_store() }
```

Comment:
We're not using this string or view anymore but deleting this string caused compile issues so decided to change it.